docs/content/overview.md

```diff
@@ -344,6 +345,7 @@ operations more efficient.
 | QingStor                     | No    | Yes  | No   | No      | Yes     | Yes   | No           | No [#2178](https://github.com/rclone/rclone/issues/2178) | No  | No |
 | SFTP                         | No    | No   | Yes  | Yes     | No      | No    | Yes          | No [#2178](https://github.com/rclone/rclone/issues/2178) | Yes  | Yes |
 | SugarSync                    | Yes   | Yes  | Yes  | Yes     | No      | No    | Yes          | Yes         | No  | Yes |
+| Tardigrade                   | Yes † | No   | No   | No      | No      | Yes   | Yes          | No          | No  | No  |
```

Comment:
Note that without Move or Copy users won't be able to use the `--backup-dir` functionality of rclone. Almost all the backends support either Move or Copy.

Maybe something for a future improvements to the protocol as I doubt it is easy!

Reply:
Good point! We plan to implement server-side Copy and Move in Tardigrade, but we are not ready with this yet. This feature is coming in the next months. Until then we will have this limitation.